utilities/xmlrpcpp/src/XmlRpcUtil.cpp

```diff
@@ -126,12 +130,16 @@ XmlRpcUtil::parseTag(const char* tag, std::string const& xml, int* offset)
 bool 
 XmlRpcUtil::findTag(const char* tag, std::string const& xml, int* offset)
 {
+  if (xml.length() > __INT_MAX__) return false;
   if (*offset >= int(xml.length())) return false;
   size_t istart = xml.find(tag, *offset);
   if (istart == std::string::npos)
     return false;
+  size_t iend = istart + strlen(tag);
+  if ((iend > __INT_MAX__)
```

Comment:
Two parens here, I doubt this compiles, double check that and any tests.

Reply:
Updated with 7d51a7aaea0cf022cc82357e03db6340741364d3